### PR TITLE
MET-8: http runtime name clash

### PR DESCRIPTION
**Pull request by afmika**

### CODE REVIEW

**Review by zifeo — DISMISSED** [review]

**Review by msga-mmm — APPROVED** [review]
It looks good to me.

**Review by Natoandro — CHANGES_REQUESTED** [review]
See https://metatypedev.slack.com/archives/C03J804A7TQ/p1677264476707969

File: typegraph/typegraph/runtimes/http.py [hunk 1]
```diff
@@ -85,6 +85,7 @@ class RESTMat(Materializer):
     content_type: str = field(kw_only=True, default="application/json")
     header_prefix: Optional[str] = "header#"
     query_fields: Optional[Tuple[str, ...]] = field(kw_only=True, default=None)
+    rename_fields: Optional[Tuple[Tuple]] = field(kw_only=True, default=None)
```
Comment (Natoandro) [inline typegraph/typegraph/runtimes/http.py:88]: `Optional[Tuple[Tuple[str, str]]]`?

**Review by Natoandro — DISMISSED** [review]

File: typegraph/typegraph/runtimes/http.py (hunk 1 above)
Comment (Natoandro) [reply typegraph/typegraph/runtimes/http.py:88]: I think a `dict` would be easier to write; and will not produce this kind of mistake: `(("field", "renamed"))`.
You can set `field(converter=frozendict)` to automatically transform it into `frozendict`.

Comment (afmika) [reply typegraph/typegraph/runtimes/http.py:88]: You are right, I had errors with a raw dict as it is not hashable so I used tuple as a proof of concept. Thanks, I will try that.

**Pull request merged**